smart-views/calendar/component.js

```diff
@@ -16,13 +17,14 @@ export default Ember.Component.extend(SmartViewMixin.default, {
         this.sendAction('updateRecordPerPage');
       }
 
+      that.set('calendarId', `${this.get('element.id')}-calendar`);
       Ember.$.getScript('//cdnjs.cloudflare.com/ajax/libs/fullcalendar/3.1.0/fullcalendar.min.js', function () {
         that.set('loaded', true);
 
-        $('#calendar').fullCalendar({
+        $(`#${that.get('calendarId')}`).fullCalendar({
           allDaySlot: false,
           minTime: '00:00:00',
-          defaultDate: new Date(2018, 2, 1),
+          defaultDate: new Date(2018, 5, 1),
```

Comment:
By default, the date is `today`. We hardcode this date (March 2018) to have enough data when using our Live Demo. Can you revert this change?